pkgs/applications/virtualization/ovmfvartool/default.nix

```
@@ -1,28 +1,17 @@
-{ lib, stdenvNoCC, fetchFromGitHub, python3 }:
+{ lib, buildPythonPackage, fetchFromGitHub, pyyaml, fetchpatch }:
```

fetchpatch is unused
```suggestion
{ lib, buildPythonPackage, fetchFromGitHub, pyyaml }:
```